app/views/notes/_note.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="card">
+  <h5 class="card-header">Created at: <%= note.created_at.to_date.strftime("%d %b, %Y") %></h5>
+  <div class="card-body">
+    <h5 class="card-title"><b>Title:</b> <%= note.title %></h5>
+    <p class="card-text"><b>Note:</b> <%= note.body %></p>
```

Comment:
- missing spaces
- truncate `body`

Reply:
- spaces done.

- it's individual show (details) view.